integr8ly/integreatly-operator
# Intly 1555

Addresses: 
- https://issues.jboss.org/browse/INTLY-1798
- https://issues.jboss.org/browse/INTLY-1799
- https://issues.jboss.org/browse/INTLY-1555

Image is built and pushed to:
quay.io/integreatly/integreatly-operator:v1.0.0

requires:
https://github.com/integr8ly/operator-registry/pull/4

Verification:
- create integreatly-operator namespace
- Add operator-registry catalog source to that namespace
- Create an operatorgroup in that namespace
- Subscribe to the integreatly-operator
- Create an installation CR
- After enough time, amq streams should be running in the openshift-amq-streams namespace.
